pyproject.toml

```diff
@@ -302,12 +302,13 @@ deps =
     jupytext
     nbconvert
     leidenalg
+    squidpy
+    pot
```

Comment:
why do we need this?

Reply:
exactly, installation of squidpy is also in the [spatial] optional requirements

Reply:
For the tutorials folder. If we don't need to check them then I can remove.